src/A0_initialization.py

```diff
@@ -321,6 +336,14 @@ def process_user_arguments(
             {"path_pdf_report": os.path.join(path_output_folder, PDF_REPORT)}
         )
 
+    if save_png is True:
+        user_input.update({"path_png_figs": path_output_folder})
+
+    if pdf_report is True and save_png is True:
+        raise ValueError(
+            "The option '-pdf' cannot be used conjointly with the option '-png'"
```

Comment:
Already discussed this, probably you are enabling conjoint execution in a later commit